pom.xml

```diff
@@ -52,6 +52,14 @@
                 </exclusion>
             </exclusions>
         </dependency>
+
+        <dependency>
+            <groupId>com.github.sheigutn</groupId>
+            <artifactId>pushbullet-java-8</artifactId>
+            <version>1.3.4</version>
+            <scope>provided</scope>
```

Comment:
We'll need to figure a way to let TravisCI also download this library, so it won't have compile errors